### DISCUSSION TITLE
launch after updating to version: 1.2.0

### DISCUSSION THREAD

**Discussion by PaulRohlik** [frangoteam/FUXA#1444] [Q&A]
![launch after updating to version-1 2 0](https://github.com/user-attachments/assets/e224e0c6-a31e-4f97-928c-b7591cc55dea)

Hi,
I'm still struggling with updates. Currently in version 1.2.0, see image. Can someone please advise what to do with it? I apologize. I'm not a programmer. User only

**Comment by unocelli** [comment]
Hi, I suppose that you have to check the quotes in your script

**Comment by PaulRohlik** [comment]
Hi,

previous problem solved. The error was in my script in the fuxa editor.

But when I try to install fuxa with the standard command:
"npm install -g --unsafe-perm @frangoteam/fuxa"

it shows me a lot of errors that I don't know how to deal with :/
node -v
v18.20.4
npm -v
10.9.0
debug version
4.3.7



C:\Users\latabpc014>npm install -g --unsafe-perm @frangoteam/fuxa
npm warn deprecated @npmcli/move-file@1.1.2: This functionality has been moved to @npmcli/fs
npm warn deprecated inflight@1.0.6: This module is not supported, and leaks memory. Do not use it. Check out lru-cache if you want a good and tested way to coalesce async requests by a key value, which is much more comprehensive and powerful.
npm warn deprecated npmlog@5.0.1: This package is no longer supported.
npm warn deprecated npmlog@6.0.2: This package is no longer supported.
npm warn deprecated npmlog@4.1.2: This package is no longer supported.
npm warn deprecated rimraf@3.0.2: Rimraf versions prior to v4 are no longer supported
npm warn deprecated rimraf@3.0.2: Rimraf versions prior to v4 are no longer supported
npm warn deprecated rimraf@3.0.2: Rimraf versions prior to v4 are no longer supported
npm warn deprecated rimraf@3.0.2: Rimraf versions prior to v4 are no longer supported
npm warn deprecated glob@7.2.3: Glob versions prior to v9 are no longer supported
npm warn deprecated are-we-there-yet@3.0.1: This package is no longer supported.
npm warn deprecated are-we-there-yet@2.0.0: This package is no longer supported.
npm warn deprecated are-we-there-yet@1.1.7: This package is no longer supported.
npm warn deprecated debug@4.1.1: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm warn deprecated debug@4.1.1: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm warn deprecated debug@4.1.1: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm warn deprecated gauge@4.0.4: This package is no longer supported.
npm warn deprecated gauge@3.0.2: This package is no longer supported.
npm warn deprecated gauge@2.7.4: This package is no longer supported.
npm warn cleanup Failed to remove some directories [
npm warn cleanup   [
npm warn cleanup     'C:\\Users\\latabpc014\\AppData\\Roaming\\npm\\node_modules\\@frangoteam\\fuxa\\node_modules',
npm warn cleanup     [Error: EPERM: operation not permitted, rmdir 'C:\Users\latabpc014\AppData\Roaming\npm\node_modules\@frangoteam\fuxa\node_modules\@mapbox\node-pre-gyp\lib\util\nw-pre-gyp'] {
npm warn cleanup       errno: -4048,
npm warn cleanup       code: 'EPERM',
npm warn cleanup       syscall: 'rmdir',
npm warn cleanup       path: 'C:\\Users\\latabpc014\\AppData\\Roaming\\npm\\node_modules\\@frangoteam\\fuxa\\node_modules\\@mapbox\\node-pre-gyp\\lib\\util\\nw-pre-gyp'
npm warn cleanup     }
npm warn cleanup   ],
npm warn cleanup   [
npm warn cleanup     'C:\\Users\\latabpc014\\AppData\\Roaming\\npm\\node_modules\\@frangoteam\\fuxa',
npm warn cleanup     [Error: EPERM: operation not permitted, rmdir 'C:\Users\latabpc014\AppData\Roaming\npm\node_modules\@frangoteam\fuxa\node_modules\node-gyp\test\fixtures'] {
npm warn cleanup       errno: -4048,
npm warn cleanup       code: 'EPERM',
npm warn cleanup       syscall: 'rmdir',
npm warn cleanup       path: 'C:\\Users\\latabpc014\\AppData\\Roaming\\npm\\node_modules\\@frangoteam\\fuxa\\node_modules\\node-gyp\\test\\fixtures'
npm warn cleanup     }
npm warn cleanup   ],
npm warn cleanup   [
npm warn cleanup     'C:\\Users\\latabpc014\\AppData\\Roaming\\npm\\node_modules\\@frangoteam\\fuxa\\node_modules\\node-gyp',
npm warn cleanup     [Error: EPERM: operation not permitted, rmdir 'C:\Users\latabpc014\AppData\Roaming\npm\node_modules\@frangoteam\fuxa\node_modules\node-gyp'] {
npm warn cleanup       errno: -4048,
npm warn cleanup       code: 'EPERM',
npm warn cleanup       syscall: 'rmdir',
npm warn cleanup       path: 'C:\\Users\\latabpc014\\AppData\\Roaming\\npm\\node_modules\\@frangoteam\\fuxa\\node_modules\\node-gyp'
npm warn cleanup     }
npm warn cleanup   ],
npm warn cleanup   [
npm warn cleanup     'C:\\Users\\latabpc014\\AppData\\Roaming\\npm\\node_modules\\@frangoteam\\fuxa\\node_modules\\winston-transport',
npm warn cleanup     [Error: EPERM: operation not permitted, rmdir 'C:\Users\latabpc014\AppData\Roaming\npm\node_modules\@frangoteam\fuxa\node_modules\winston-transport'] {
npm warn cleanup       errno: -4048,
npm warn cleanup       code: 'EPERM',
npm warn cleanup       syscall: 'rmdir',
npm warn cleanup       path: 'C:\\Users\\latabpc014\\AppData\\Roaming\\npm\\node_modules\\@frangoteam\\fuxa\\node_modules\\winston-transport'
npm warn cleanup     }
npm warn cleanup   ]
npm warn cleanup ]
npm error code 1
npm error path C:\Users\latabpc014\AppData\Roaming\npm\node_modules\@frangoteam\fuxa\node_modules\node-snap7
npm error command failed
npm error command C:\Windows\system32\cmd.exe /d /s /c prebuild-install || node-gyp rebuild
npm error prebuild-install WARN install No prebuilt binaries found (target=18.20.4 runtime=node arch=x64 libc= platform=win32)
npm error gyp info it worked if it ends with ok
npm error gyp info using node-gyp@8.4.1
npm error gyp info using node@18.20.4 | win32 | x64
npm error gyp info find Python using Python version 3.13.0 found at "C:\Users\latabpc014\AppData\Local\Programs\Python\Python313\python.exe"
npm error gyp info find VS using VS2019 (16.0.28729.10) found at:
npm error gyp info find VS "C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools"
npm error gyp info find VS run with --verbose for detailed information
npm error gyp info spawn C:\Users\latabpc014\AppData\Local\Programs\Python\Python313\python.exe
npm error gyp info spawn args [
npm error gyp info spawn args   'C:\\Users\\latabpc014\\AppData\\Roaming\\npm\\node_modules\\@frangoteam\\fuxa\\node_modules\\node-gyp\\gyp\\gyp_main.py',
npm error gyp info spawn args   'binding.gyp',
npm error gyp info spawn args   '-f',
npm error gyp info spawn args   'msvs',
npm error gyp info spawn args   '-I',
npm error gyp info spawn args   'C:\\Users\\latabpc014\\AppData\\Roaming\\npm\\node_modules\\@frangoteam\\fuxa\\node_modules\\node-snap7\\build\\config.gypi',
npm error gyp info spawn args   '-I',
npm error gyp info spawn args   'C:\\Users\\latabpc014\\AppData\\Roaming\\npm\\node_modules\\@frangoteam\\fuxa\\node_modules\\node-gyp\\addon.gypi',
npm error gyp info spawn args   '-I',
npm error gyp info spawn args   'C:\\Users\\latabpc014\\AppData\\Local\\node-gyp\\Cache\\18.20.4\\include\\node\\common.gypi',
npm error gyp info spawn args   '-Dlibrary=shared_library',
npm error gyp info spawn args   '-Dvisibility=default',
npm error gyp info spawn args   '-Dnode_root_dir=C:\\Users\\latabpc014\\AppData\\Local\\node-gyp\\Cache\\18.20.4',
npm error gyp info spawn args   '-Dnode_gyp_dir=C:\\Users\\latabpc014\\AppData\\Roaming\\npm\\node_modules\\@frangoteam\\fuxa\\node_modules\\node-gyp',
npm error gyp info spawn args   '-Dnode_lib_file=C:\\\\Users\\\\latabpc014\\\\AppData\\\\Local\\\\node-gyp\\\\Cache\\\\18.20.4\\\\<(target_arch)\\\\node.lib',
npm error gyp info spawn args   '-Dmodule_root_dir=C:\\Users\\latabpc014\\AppData\\Roaming\\npm\\node_modules\\@frangoteam\\fuxa\\node_modules\\node-snap7',
npm error gyp info spawn args   '-Dnode_engine=v8',
npm error gyp info spawn args   '--depth=.',
npm error gyp info spawn args   '--no-parallel',
npm error gyp info spawn args   '--generator-output',
npm error gyp info spawn args   'C:\\Users\\latabpc014\\AppData\\Roaming\\npm\\node_modules\\@frangoteam\\fuxa\\node_modules\\node-snap7\\build',
npm error gyp info spawn args   '-Goutput_dir=.'
npm error gyp info spawn args ]
npm error Traceback (most recent call last):
npm error   File "C:\Users\latabpc014\AppData\Roaming\npm\node_modules\@frangoteam\fuxa\node_modules\node-gyp\gyp\gyp_main.py", line 42, in <module>
npm error     import gyp  # noqa: E402
npm error     ^^^^^^^^^^
npm error   File "C:\Users\latabpc014\AppData\Roaming\npm\node_modules\@frangoteam\fuxa\node_modules\node-gyp\gyp\pylib\gyp\__init__.py", line 9, in <module>
npm error     import gyp.input
npm error   File "C:\Users\latabpc014\AppData\Roaming\npm\node_modules\@frangoteam\fuxa\node_modules\node-gyp\gyp\pylib\gyp\input.py", line 19, in <module>
npm error     from distutils.version import StrictVersion
npm error ModuleNotFoundError: No module named 'distutils'
npm error gyp ERR! configure error
npm error gyp ERR! stack Error: `gyp` failed with exit code: 1
npm error gyp ERR! stack     at ChildProcess.onCpExit (C:\Users\latabpc014\AppData\Roaming\npm\node_modules\@frangoteam\fuxa\node_modules\node-gyp\lib\configure.js:259:16)
npm error gyp ERR! stack     at ChildProcess.emit (node:events:517:28)
npm error gyp ERR! stack     at ChildProcess._handle.onexit (node:internal/child_process:292:12)
npm error gyp ERR! System Windows_NT 10.0.19045
npm error gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\latabpc014\\AppData\\Roaming\\npm\\node_modules\\@frangoteam\\fuxa\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"
npm error gyp ERR! cwd C:\Users\latabpc014\AppData\Roaming\npm\node_modules\@frangoteam\fuxa\node_modules\node-snap7
npm error gyp ERR! node -v v18.20.4
npm error gyp ERR! node-gyp -v v8.4.1
npm error gyp ERR! not ok
npm error A complete log of this run can be found in: C:\Users\latabpc014\AppData\Local\npm-cache\_logs\2024-10-30T08_06_01_662Z-debug-0.log

C:\Users\latabpc014>

**Reply by unocelli** [reply]
Hi, I suppose that you have a problem with the installation of node-snap7 library, please tra the [NPM @frangoteam/fuxa-min](https://www.npmjs.com/package/@frangoteam/fuxa-min) version:
npm install -g --unsafe-perm @frangoteam/fuxa-min

**Comment by PaulRohlik** [comment]
Hello,
Thank you for your reply. It runs well, but I have discovered a new bug/complication.
When I would like to add a functionality trough the Tag selection, the new windows will open – let´s have a look on my screenshot.
We have over 600 lines and in previous version there were a possibility to seek trough the folders Name, Address or Device asw.
Now when you write any keyword into these folders (bove in my screenshot), it will not seek for the words in the list.
It looks like a bug in the program. Do you have some advice to let it work again? Thank you, ….
![image](https://github.com/user-attachments/assets/ada5dff2-9ed7-4fe7-bbdd-73f91a155b08)

**Comment by unocelli** [comment]
Hi, have you tried the latest version 1.2.1?

**Comment by PaulRohlik** [comment]
![image](https://github.com/user-attachments/assets/588722e5-c19a-40f1-becc-6777d054f8c4)
hi, yes yes

**Comment by PaulRohlik** [comment]
hi,
I tried uninstalling and reinstalling but the search in the TAG list is not working. Please help. Where are all the FUXA data stored? I would try to delete them manually. I am using win 10

**Reply by unocelli** [reply]
I will install the last one version 1.2.3

**Reply by PaulRohlik** [reply]
hi,
yes. I installed the latest version. 
The problem will be in the host PC. 
When I use the command: "npm uninstall -g --unsafe-perm @frangoteam/fuxa-min" what files will remain on the PC?

I use:
npm 10.9.0
node v18.20.4

**Comment by unocelli** [comment]
Hi, in FUXA/server/_appdata you will find your project, copy the folder

**Comment by PaulRohlik** [comment]
Hello,
after the reinstallation of Fuxa version I didn´t achieved any improvement and I think, we have already checked many possibilities how to restore it. We tried also the installation on another computer and the search of Tag list dind´t worked there neither. Would it be, please, possibility to send you my project to check it at your side, if the project /search in TAG List/ would run under your program or to see, If there is somenthing "broken", even if there should be some fee related to it? Best Regars,

**Reply by unocelli** [reply]
Yes, please you can send the project to 4frango@gmail.com

**Reply by unocelli** [reply]
Hi, I think that I found the issue — in your project, there are some tags configured without a name.  
The filter logic should now be more robust and handle this case properly.  
Please try again with the latest version on the `master` branch and let me know if it works for you.

**Comment by PaulRohlik** [comment]
Hi,
Installed version 1.2.5 but tag sheet is still freezing :/
---------- Původní e-mail ----------
Od: Umberto Nocelli ***@***.***>
Komu: frangoteam/FUXA ***@***.***>
Kopie: PaulRohlik ***@***.***>, Author ***@***.***>
Datum: 4. 5. 2025 21:47:55
Předmět: Re: [frangoteam/FUXA] launch after updating to version: 1.2.0 
(Discussion #1444)
"


Hi, I think that I found the issue — in your project, there are some tags 
configured without a name.
The filter logic should now be more robust and handle this case properly.
Please try again with the latest version on the master branch and let me 
know if it works for you.

—
Reply to this email directly, view it on GitHub
(https://github.com/frangoteam/FUXA/discussions/1444#discussioncomment-13032806)
, or unsubscribe
(https://github.com/notifications/unsubscribe-auth/ALTO7UHLF7UY4AZKNPKSEOL24ZVGRAVCNFSM6AAAAABQAVEJFKVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTGMBTGI4DANQ)
.
You are receiving this because you authored the thread. Message ID: <
***@***.***>
"

**Comment by PaulRohlik** [comment]
Hi,
I found individual tag and repaired the names. Everything is already 
functional. 
Thank you very much








---------- Původní e-mail ----------
Od: Umberto Nocelli ***@***.***>
Komu: frangoteam/FUXA ***@***.***>
Kopie: PaulRohlik ***@***.***>, Author ***@***.***>
Datum: 4. 5. 2025 21:47:55
Předmět: Re: [frangoteam/FUXA] launch after updating to version: 1.2.0 
(Discussion #1444)
"


Hi, I think that I found the issue — in your project, there are some tags 
configured without a name.
The filter logic should now be more robust and handle this case properly.
Please try again with the latest version on the master branch and let me 
know if it works for you.

—
Reply to this email directly, view it on GitHub
(https://github.com/frangoteam/FUXA/discussions/1444#discussioncomment-13032806)
, or unsubscribe
(https://github.com/notifications/unsubscribe-auth/ALTO7UHLF7UY4AZKNPKSEOL24ZVGRAVCNFSM6AAAAABQAVEJFKVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTGMBTGI4DANQ)
.
You are receiving this because you authored the thread. Message ID: <
***@***.***>
"
